diff --git a/src/ri_pt2/mod.rs b/src/ri_pt2/mod.rs
index ed2d1f5..c2a1fd9 100644
--- a/src/ri_pt2/mod.rs
+++ b/src/ri_pt2/mod.rs
@@ -1787,6 +1787,7 @@ pub fn restricted_open_shell_pt2_rayon(scf_data: &SCF) -> anyhow::Result<[f64;3]
                 };
                 let (sender, receiver) = channel();
                 elec_pair.par_iter().for_each_with(sender,|s,i_pair| {
+                    omp_set_num_threads_wrapper(1);
                     let mut e_mp2_term_os = 0.0_f64;
                     let i_state = i_pair[0];
                     let j_state = i_pair[1];
